omnibus/config/software/libjemalloc.rb

```diff
@@ -0,0 +1,42 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2022-present Datadog, Inc.
+
+name "libjemalloc"
+default_version "5.3.0"
+
+version "5.3.0" do
+  source sha256: "ef6f74fd45e95ee4ef7f9e19ebe5b075ca6b7fbe0140612b2a161abafb7ee179" 
+end
+
+ship_source_offer true
+
+source url:"https://github.com/jemalloc/jemalloc/archive/refs/tags/#{version}.tar.gz",
+       extract: :seven_zip
+
+relative_path "libjemalloc"
```

Comment:
This refers to the path to the source folder inside the archive, in this case I think it's:

```suggestion
relative_path "jemalloc-#{version}"
```